### PR TITLE
User Login and Register 

### PR DESCRIPTION
**Pull request by mariamconde**
The user can create an account on the frontend and sign in, both are connected to the backend.

### CODE REVIEW

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
please add some comments to the new files, it'll make the reviews quicker

**Comment by Rebecca-CYLi** [comment]
there's also some merge conflicts to fix

File: react-app/pomodoro-timer-ui/src/Components/App/App.jsx [hunk 1]
```diff
@@ -3,8 +3,13 @@ import Timer from "../Timer/Timer";
 import SidebarContainer from "../Sidebar/Sidebar";
 import './App.css'
 
+
 export default function App() {
+<<<<<<< HEAD
+  // console.clear()
+=======
   console.clear()
+>>>>>>> main
```
Comment (Rebecca-CYLi) [inline react-app/pomodoro-timer-ui/src/Components/App/App.jsx]: there's leftover merge conflicts

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
there's a merge conflict leftovers

**Review by MonCaptain — COMMENTED** [review]
changes reviewed. Nothing out of the ordinary

File: react-app/pomodoro-timer-ui/src/Components/App/App.jsx (hunk 1 above)
Comment (Rebecca-CYLi) [reply react-app/pomodoro-timer-ui/src/Components/App/App.jsx]: merge conflict resolved

**Review by MonCaptain — APPROVED** [review]

**Review by Rebecca-CYLi — APPROVED** [review]
changes made, G2G

**Pull request merged**